### PR TITLE
Create function.get_param.php

### PR DESCRIPTION
**Pull request by Roosso**
Example
`<a href="index.php?page={$page['name']}&pg={$page['n']}{get_params prefix='&amp;' exclude='page,pg'}">{$page['n']}</a>`

### CODE REVIEW

File: libs/plugins/function.get_param.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * Type:     function<br>
+ * Name:     get_params<br>
+ * Purpose:  generate get parameters
+ * Input:
+ *         - prefix  = before string
+ *         - suffix  = after string
+ *         - exclude = GET params wich should be excluded
+ *
+ * @author Jambik <> idea and creations
```
Comment (Roosso) [inline libs/plugins/function.get_param.php]: sorry...double

**Comment by AnrDaemon** [comment]
Somebody tested the "New pull request" button?
This can be closed as unrelated, IMO.

**Comment by uwetews** [comment]
This is more a user specific plugin subject for a user plugins folder

**Comment by Roosso** [comment]
I dont understand...

See:
In the template, you often have to use dynamic links
For example: /catalog.php?Param1=x&param2=y&page=2
I need to designate a link for the third page, but it also needs to list all the parameters. That would not be used each time the construction of $smarty.get.param1 easier to use the plugin, as I showed.

`<a href="index.php?{get_params prefix='&amp;' exclude='page'}&page=1">Prev</a>
<a href="index.php?{get_params prefix='&amp;' exclude='page'}&page=3">Next</a>`

**Comment by Roosso** [comment]
The documentation did not find anything about the user folder with the plugins. Maybe not looking there !?

**Comment by AnrDaemon** [comment]
Read the entire chapter III. And specifically https://www.smarty.net/docs/en/api.add.plugins.dir.tpl

**Pull request closed without merging**